devtools/conda-envs/no_openeye.yaml

```diff
@@ -1,6 +1,7 @@
 name: bespokefit-no-openeye
 
 channels:
+  - conda-forge/label/openff-interchange_rc
```

Comment:
This isn't needed for Pydantic 2 compatibility

Reply:
Gotcha, rerunning tests without it